bindings/pyroot/cppyy/CPyCppyy/include/CPyCppyy/PyException.h

```diff
@@ -39,12 +39,12 @@ namespace CPyCppyy {
 class CPYCPPYY_CLASS_EXTERN PyException : public std::exception {
 public:
     PyException();
-    virtual ~PyException() noexcept;
+    ~PyException() noexcept override;
 
-// give reason for raised exception
-    virtual const char* what() const noexcept;
+    // give reason for raised exception
+    const char *what() const noexcept override;
```

Comment:
```suggestion
    const char* what() const noexcept override;
```

Reply:
Thanks, good point. All this was clang-tidy auto-fixits messing up the formatting. Do you know if wlav has a .clang-tidy file for cppyy that we could insert to prevent or this? or just clang-format off?